app/src/main/java/com/example/testapp/ui/login/LoginActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.testapp.ui.login
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+
+
+import androidx.lifecycle.Observer
+import com.example.testapp.R
+
+class LoginActivity : AppCompatActivity() {
```

Comment:
Please make this a fragment. It is not good to have multiple activities for multiple fragments. Host these fragments in single Activity (Please name this Activity to be something meaningful)

Reply:
changed to fragment